### PR TITLE
Feat:Create CTA sections for android app development page

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [ ] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI 
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Video:**

https://github.com/canopas/canopas-website/assets/98794636/d1aae310-9017-4d6d-a1a0-06272e4250a1

**Performance desktop:**
image

**Performance Mobile:**
image

**Description:**

1. Create 3 CTA for this page.
2. Fix landing section vertical alignment text in lg.

**Desktop Images**
image
image

**Mobile Image:**
image

### CODE REVIEW

File: vue-frontend/src/components/android-app-development/BlackCtaSection.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+      </div>
+      <div class="tw-mx-auto tw-mt-4">
+        <router-link
+          class="gradient-btn consultation-btn tw-m-0 tw-mx-auto tw-flex tw-w-max tw-items-center tw-rounded-full tw-p-3 tw-text-center"
+          :to="contactURL"
+          @click.native="mixpanel.track('tap_android_development_cta')"
+        >
+          <span
+            class="tw-mr-2.5 tw-font-inter-medium tw-text-[1rem] tw-font-normal tw-leading-[1.1875rem] !tw-tracking-[0] md:tw-block md:tw-text-[1.09375rem] md:tw-leading-[1.3125rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.4375rem]"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/android-app-development/BlackCtaSection.vue:34]: Make sure all button's height should be same as header button

File: vue-frontend/src/pages/android-app-development.vue [hunk 2]
```diff
@@ -35,6 +46,7 @@ export default {
         landing: "view_android_development_landing_section",
         footer: "view_android_development_footer",
         development: "view_android_development_section",
+        cta: "view_android_development_cta_section",
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/android-app-development.vue:49]: I think event should be individual for all CTAs

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/android-app-development/BlackCtaSection.vue (hunk 1 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/android-app-development/BlackCtaSection.vue:34]: yes, its same.

File: vue-frontend/src/components/android-app-development/CtaSection.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        alt="android-cta-image"
+      />
+    </div>
+    <div
+      class="tw-border-[#00000]/[0.6] tw-absolute tw-left-[10%] tw-z-0 tw-mr-[0.2rem] tw-mt-[2rem] tw-flex tw-flex-col tw-items-center tw-justify-center tw-rounded-lg tw-border-2 tw-py-[0.5rem] md:!tw-mt-[4rem] lg:!tw-mt-[6rem] xl:!tw-mt-[8rem] xl:tw-py-[2rem] 2xl:!tw-pb-[4rem] 2xl:!tw-pt-[3rem] 2xl:tw-pr-[2rem] xs:tw-mt-[3rem] xs:tw-py-4"
+    >
+      <div
+        class="tw-ml-auto tw-w-[38%] tw-font-inter-semibold tw-text-[1.125rem] tw-leading-[1.4625rem] tw-text-black-core/[0.87] md:tw-text-[2.4375rem] md:tw-leading-[3.15625rem] lg:!tw-w-[47%] lg:tw-text-[3.4375rem] lg:tw-leading-[5.15625rem] xs:tw-w-[50%]"
+      >
+        <span>Let’s talk about what we can build and scale together.</span>
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/android-app-development/CtaSection.vue:16]: Maybe you need to make the font smaller in lg to xl.

Comment (cp-priyanka-g) [reply vue-frontend/src/components/android-app-development/CtaSection.vue:16]: its according to image size.

Comment (cp-mansi-d) [reply vue-frontend/src/components/android-app-development/CtaSection.vue:16]: But it doesn't look good see, it covers the whole box

cta1

**Review by cp-sumi-k — APPROVED** [review]

File: vue-frontend/src/components/android-app-development/CtaSection.vue (hunk 3 above)
Comment (cp-mansi-d) [reply vue-frontend/src/components/android-app-development/CtaSection.vue:16]: Also provide some line-height for lg device

**Review by cp-mansi-d — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**